__tests__/components/Toast.test.tsx

```diff
@@ -7,20 +7,91 @@ describe('Toast', () => {
     const onDismiss = jest.fn();
 
     test('should render the toast component with the message', () => {
-        render(<Toast message="This is a toast message" isVisible={true} timeToShow={5000} onDismiss={onDismiss} />);
+        render(
+            <Toast
+                message="This is a toast message"
+                isVisible={true}
+                timeToShow={5000}
+                onDismiss={onDismiss}
+                type="success"
+            />
+        );
 
         expect(screen.getByText('This is a toast message')).toBeInTheDocument();
+        expect(screen.getByTestId('toast')).toBeInTheDocument();
```

Comment:
I think this check should be done before checking the toast message. i.e the previous line should be after the current line.

Reply:
Fixed